how-to/software/kernel-crash-dump.md

```diff
@@ -20,12 +20,30 @@ For some of these events (kernel panic, NMI) the kernel will react automatically
 
 When a kernel panic occurs, the kernel relies on the *kexec* mechanism to quickly reboot a new instance of the kernel in a pre-reserved section of memory that had been allocated when the system booted (see below). This permits the existing memory area to remain untouched in order to safely copy its contents to storage.
 
+## KDump enabled by default
+
+Starting in Oracular Oriole (24.10) the kernel crash dump facility may be enabled by default at install time of standard installs of Ubuntu Desktop or Ubuntu Server that meet the following requirements:
```

Comment:
Changing the "may" to "can" isn't quite right, as there is no option to enable it or disable it, and using "can" implies to me that it is optional.  I wrote "may" originally to indicate that it will be enabled by default if the conditions were met, but if some edge cases I did not enumerate were not met, it wouldn't be enabled.  I would be fine with changing it to "will".

> _Starting in Oracular Oriole (24.10) the kernel crash dump facility **will** be enabled by default during standard Ubuntu Desktop or Ubuntu Server installations on systems that meet the following requirements:_

would probably be best.

Reply:
That sounds good to me :)

Reply:
Fixed in latest commit